templates/akeneo/parametersdist.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/app/config/parameters.yml.dist b/app/config/parameters.yml.dist
+index 7811663..7a1bed7 100644
+--- a/app/config/parameters.yml.dist
++++ b/app/config/parameters.yml.dist
+@@ -1,3 +1,6 @@
++imports:
++    - { resource: parameters_platform.php }
++
+ parameters:
+     database_driver:                      pdo_mysql
+     database_host:                        localhost
+@@ -11,3 +14,5 @@ parameters:
+     product_model_index_name:             akeneo_pim_product_model
+     product_and_product_model_index_name: akeneo_pim_product_and_product_model
+     index_hosts:                          'localhost: 9200'
++    mailer_host:                          "%env(PLATFORM_SMTP_HOST)%"
++    installer_data:                       PimInstallerBundle:minimal
```

Comment:
Aren't there some removals now missing from this file?

Reply:
Yes, I streamlined it down. Some where duplicates with the Akeneo defaults anyways